include/v8-profiler.h

```diff
@@ -175,6 +180,19 @@ class V8_EXPORT CpuProfile {
 
 
 /**
+ * HeapEventXDK contains the latest chunk of heap info
+ */
+class V8_EXPORT HeapEventXDK {
+ public:
+  const char* getSymbols();
```

Comment:
Currently this API is called from WebKit, the content of the string is written to stream and sent to frontrend.

During this operation the original strings are owned by the HeapEventXDK, which cannot be destroyed.